.github/workflows/test.yml

```
@@ -225,7 +225,7 @@ jobs:
           path: node_modules/.cache
           key: ${{ runner.os }}-cloud-jest-${{ github.run_id }}
           restore-keys: ${{ runner.os }}-cloud-jest-
-      - run: yarn test:cloud --coverage --maxWorkers=100%
+      - run: yarn test:cloud --coverage --maxWorkers=80%
```

because the other jest invocation uses 100%, you should justify this with a comment.
also, instead of 80%, and knowing that it competes with the server, can you be explicit and use `=1` or just omit it (which is equivalent to ["the number of cores available on your machine minus one" (ie `=1`)](https://jestjs.io/docs/cli#--maxworkersnumstring)?

```suggestion
      # Only use 1 worker, so the other can be used for the proxy server under test.
      - run: yarn test:cloud --coverage
```